frontend/src/App.vue

```diff
@@ -73,9 +44,10 @@ const isAuthenticated = computed(() => currentUser?.exists());
 const call = createFetch({
   baseUrl: apiUrl,
   options: {
-    async beforeFetch({ options }) {
+    beforeFetch({ options }) {
       if (isAuthenticated.value) {
-        const token = await currentUser.data.accessToken;
+        const token = currentUser.data.accessToken;
+        // @ts-ignore
         options.headers.Authorization = `Bearer ${token}`;
```

Comment:
Sorry, my first ts-ignore 🙈 Somehow `Authorization` isn't recognized here as valid header option and I didn't found a way to properly type that within reason.